packages/preview/cram-snap/0.2.2/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Kamack38
```

Comment:
```suggestion
Copyright (c) 2024-2025 Kamack38
```

Reply:
I've read somewhere that it doesn't matter. Does it?

Reply:
I don't think it is that bad if it is outdated (or if it is not your legal name for that matter), but I believe it's preferable to keep it up-to-date. If you prefer to keep it as is, I won't block merging for that. If you remember where you read that I would be interested by a link though.